### PR TITLE
Flake8: remove E126

### PR DESCRIPTION
**Pull request by elacuesta**
_Continuation line over-indented for hanging indent_

~Based on #4542 to avoid merge conflicts.~ (merged)

### CODE REVIEW

File: scrapy/spiderloader.py [hunk 1]
```diff
@@ -24,15 +24,17 @@ def __init__(self, settings):
...
-                           for (mod, cls) in locations)
-                 for name, locations in self._found.items()
-                 if len(locations) > 1]
+        dupes = []
+        for name, locations in self._found.items():
+            dupes.extend([
+                "  {cls} named {name!r} (in {module})".format(module=mod, cls=cls, name=name)
+                for mod, cls in locations
+            ])
+
```
Comment (elacuesta) [inline scrapy/spiderloader.py:33]: This might be a little bit more verbose, but way more readable IMHO. Personally, I find nested `for` lops hard to follow.

Comment (kmike) [reply scrapy/spiderloader.py:33]: It seems `if len(locations) > 1` check is removed in the new code. How does it work? Do we have tests for it?

Comment (elacuesta) [reply scrapy/spiderloader.py:33]: Thanks for noticing this. I thought the list would be empty in that case, but it's `>1` and not `>=1`. I'll fix it.

Comment (kmike) [reply scrapy/spiderloader.py]: I was thinking that's the purpose of the function - warn in case of duplicates, i.e. a single name with 2+ locations; that's bad that our tests don't catch it.

Comment (elacuesta) [reply scrapy/spiderloader.py]: Makes sense. I'll open a issue to keep track of this, I don't think we should add it in this PR.

Comment (elacuesta) [reply scrapy/spiderloader.py]: Done: #4549. I skipped the issue template entirely :sweat_smile:
